bin/update-codegen.sh

```diff
@@ -10,7 +10,7 @@ codegen_pkg=${GOPATH}/pkg/mod/k8s.io/code-generator@${gen_ver}
 # ROOT_PACKAGE :: the package that is the target for code generation
 ROOT_PACKAGE=github.com/linkerd/linkerd2
 
-crds=(serviceprofile:v1alpha2 server:v1beta1 serverauthorization:v1beta1 link:v1alpha1 policy:v1alpha1)
+crds=(serviceprofile:v1alpha2 server:v1beta1 serverauthorization:v1beta1 link:v1alpha1 policy:v1alpha1 policy:v1beta3)
```

Comment:
I think it should be safe to just bump the version and get rid of all the `policy/v1alpha1` directories.
```suggestion
crds=(serviceprofile:v1alpha2 server:v1beta1 serverauthorization:v1beta1 link:v1alpha1 policy:v1beta3)
```

Reply:
I think AuthorizationPolicy, MeshTLSAuthentication, and NetworkAuthentication are all in `policy/v1alpha1`, aren't they?

Reply:
You're right, I was having tunnel vision on just HTTPRoute.